aries-site/src/pages/components/all-components.mdx

```diff
@@ -553,14 +553,14 @@ Jump to a specific component type:
   <SpinnerExample />
 </Example>
 
-### Table
```

Comment:
For the all components page, does it make sense to have a "Table" represented as well? Are there use cases where Table is favorable to DataTable?

Reply:
I am fine with that. Just thinking through degree of parity we want with Grommet.